extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/providers/DsDeviceThingTypeProvider.java

```diff
@@ -62,13 +63,13 @@
      * @author Matthias Siegele - initial contributer
      */
     public static enum SupportedThingTypes {
-        // ThingType, responsible ThingHanlder, Device config-description with power-sensors
-        GE(DeviceHandler.class.getSimpleName(), true),
-        GR(DeviceHandler.class.getSimpleName(), false),
-        SW(DeviceHandler.class.getSimpleName(), true),
-        BL(DeviceHandler.class.getSimpleName(), true),
-        dSiSens200(DeviceHandler.class.getSimpleName(), false),
-        circuit(CircuitHandler.class.getSimpleName(), false);
+    // ThingType, responsible ThingHanlder, Device config-description with power-sensors
+    GE(DeviceHandler.class.getSimpleName(), true),
```

Comment:
this is an Eclipse formatter bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=536322 which already hit us in another situation. Don't know if you want to workaround it somehow?